### ISSUE TITLE
Change Service Account Enhancement

### ISSUE THREAD

**Issue by HealisticEngineer** [Microsoft/ReportingServicesTools#150]
*feature*
Maybe this is already being worked on but including the option to change service account would be nice.

I already do this using the following, perhaps this can be added to module.
```
$RSObject = Get-WmiObject -namespace "root\Microsoft\SqlServer\ReportServer\RS_PBIRS\v14\Admin" -class "MSReportServer_ConfigurationSetting"
$ServiceAccount = "domain\account"
$useBuiltInServiceAccount = $false
$password = "superpassword"
$RSObject.SetWindowsServiceIdentity($useBuiltInServiceAccount, $builtInServiceAccount, $password) | out-null
```

**Comment by jtarquino** [comment]
Sounds like a good idea for a new function, I would like to invite you to contribute with it and send a pull request.
Let me know if you need any help with it, a good start is the [readme](https://github.com/Microsoft/ReportingServicesTools/blob/master/README.md#local-testing-and-development)

**Comment by jzabroski** [comment]
This is a great idea; please ensure password is not a required field for shops like ours that use Group MSAs.

**Comment by jzabroski** [comment]
On second thought, shouldn't this be addressed via SqlServerDsc PowerShell module? See this folder for the current examples supported by that Microsoft project: https://github.com/PowerShell/SqlServerDsc/tree/dev/Examples/Resources/SqlRS

It's possible ReportingServicesTools could help SqlServerDsc as well.

I just ask, because I have found that writing PowerShell scripts to custom configure Reporting Services to be a HUGE pain in the butt! And ReportingServicesTools hasn't really made my life that much better.  Stuff like configuring SMTP Folder Drop locations, setting up multiple instances of Reporting Services on the same box, etc.  With SqlServerDsc, they appear to have a recipe we can build off of.

**Comment by HealisticEngineer** [comment]
I agree should be in the SqlRs but with the split between SSRS and normal SQL Media, it's hard to say.
However yes we should look a perhaps making this DSC resource going forward.

**Comment by HealisticEngineer** [comment]
@jtarquino I've finally found some time to sit down and write the function, but I can't publish the branch.
permissions and all that ;)

**Comment by jtarquino** [comment]
Hi @HealisticEngineer , the best way to do it is with a fork (https://help.github.com/articles/fork-a-repo/) and then you can send a Pull Request
